site/docs/windows.md

```diff
@@ -102,6 +106,20 @@ To build C++ targets, you need:
     set BAZEL_VC=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC
     ```
 
+    and (to specify exact VC++ version)
+
+     ```
+    set BAZEL_VC_TOOL=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\bin\Hostx64\x64
+    ```
+
+    The last command sets the path to Visual C++ tools.
+
+    If you have multiple VC++ versions installed, but you did not set `BAZEL_VC_TOOL` bazel will use latest VC++ version.
+
+    If you set `BAZEL_VC_TOOL` but you did not set any of `BAZEL_VS` or `BAZEL_VS` bazel will only search
```

Comment:
Comma after `BAZEL_VS`.

Capitalize "Bazel."

"search for the Visual C++ directory based on the path provided"